src/pages/Wrapper/Wrapper.js

```diff
@@ -0,0 +1,17 @@
+import { useContext } from 'react';
+import AuthPage from '../AuthForm/AuthPage';
+import AuthContext from '../../store/auth-context';
+
+
+function Wrapper(props) {
+    const authCtx = useContext(AuthContext);
+    return(
+        <div>
+            {authCtx.isLoggedIn &&(props.children)}
```

Comment:
@AryaPV1234 This can be changes to a ternery operator to render props.children when the expression is true and AuthPage when it is false.

Reply:
Changed